posix.mak

```diff
@@ -76,6 +76,7 @@ endif
 # Configurable stuff that's rarely edited
 INSTALL_DIR = ../install
 DRUNTIME_PATH = ../druntime
+DLANG_ORG_DIR = ../dlang.org
```

Comment:
Maybe at some point in the future we get https://github.com/dlang/dmd/pull/7476 ...